DESCRIPTION

```diff
@@ -37,6 +37,8 @@ Suggests:
 VignetteBuilder: 
     knitr
 ByteCompile: true
+Config/Needs/check:
+    r-dbi/DBI
```

Comment:
Do we really need this long-term? I can see how, today, it is how we're getting the DBI dev candidate. But going forward `DBI (>= 1.2.0)` in `Suggests` is, I think, all we need.

Reply:
Thanks. Fully agree.

DBI 1.2.0 is in CRAN's queue, it needs manual intervention but should come through tomorrow.